packages/sdk/src/react/StoryBoost/StoryBoost.tsx

```diff
@@ -109,16 +109,12 @@ export class StoryBoost extends React.Component<StoryBoostProps, StoryBoostState
                 <span>Support this newsroom</span>
                 <PaymentButton onClick={this.handleStartPayment} />
               </StoryBoostFooter>
-              <PaymentsModal open={this.state.paymentsOpen}>
+              <PaymentsModal open={this.state.paymentsOpen} handleClose={this.handleClose}>
```

Comment:
You could use `handleEndPayment` for this or `handleClose` for both.

Reply:
Oh right, whoops, I didn't look at how `handleEndPayment` was being used and thought it'd be safer to keep "close" separate from "end payment" in case one had to change, but uh yeah now I see.